samples/package.json

```diff
@@ -13,7 +13,12 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@google-cloud/security-center": "^3.0.1"
+    "@google-cloud/pubsub": "^1.5.0",
+    "@google-cloud/security-center": "^3.0.1",
+    "google-gax": "^1.14.1",
```

Comment:
You should not need `google-gax`, `google-proto-files`, `path`, or `protobufjs` as imports here

Reply:
Done.